contact.html

```diff
@@ -0,0 +1,73 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Home</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/contact.css">
+
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav>
+                <img src="img/lambda-black.png" class="logo" alt="lambda logo">
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="products.html">Products</a>
+                <a href="blog.html">Blog</a>
+                <a href="contact.html">Contact</a>
+            </nav>
+        </header>
+   
+        <section class="top-content">
+                  <img class="contactImg" src="https://images.unsplash.com/uploads/1413222992504f1b734a6/1928e537?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80" alt="old rotarty phones">
+                  <div class="contact">
+                    <h1>Contact</h1>
+                
+                    <p>
+                      123123 Street Drive<br>
+                      City, State 12345<br>
+                      email@email.email<br>
+                      (123) 123-1234<br>
+                      (fax) fax-faxx
+                    </p>
+                  </div>
+                </section>
+        
+        <div class="outterForm">
+              <h1>Questions Comments Complaints</h1>
+              <form id="costomerContact">
```

Comment:
Curious why you decided to use an id instead of a class for customerContact. I noticed it was not used in your css